asapdiscovery-cli/asapdiscovery/cli/cli_args.py

```diff
@@ -87,6 +87,15 @@ def output_dir(func):
     )(func)
 
 
+def overwrite(func):
```

Comment:
It I think overwrite should still be the default, as a mixed run directory is prone to unexpected results. You can use one of the `--overwrite/--no-overwrite`  mixed binary flags.

Reply:
@apayne97 I still think overwriting should be the default, as we have been relying on this behaviour for all our uploads etc. 

There are all kinds of issues with partial writes eg a number of files that get written to a specific filename each time. eg, every time the workflow runs a bunch of CSV files are written with names like `postera_uploaded_final.csv`. If the docking workflow is run twice in the same directory this file will not reflect full state of the run. Same goes for a bunch of other files. Lots of potential for confusion.

 Unless these are read and merged or some other workaround is used I think overwrriting should be the default still. I am fine if people want to enter the danger zone themselves, but I don't think all the project support workflows should not-overwrite.

Reply:
yep, this makes sense, thanks for the longer explanation. I'll make overwriting be the default!